pandora-client-web/src/components/wardrobe/views/wardrobeItemView.tsx

```diff
@@ -252,10 +252,18 @@ function InventoryItemViewList({ item, selected = false, setFocus, singleItemCon
 	setFocus?: (newFocus: WardrobeFocus) => void;
 	singleItemContainer?: boolean;
 }): ReactElement {
-	const { targetSelector, target, extraItemActions, heldItem, setHeldItem } = useWardrobeContext();
+	const { targetSelector, target, extraItemActions, heldItem, setHeldItem, scrollToItem, setScrollToItem } = useWardrobeContext();
 	const wornItem = useWardrobeTargetItem(target, item);
 	const extraActions = useObservable(extraItemActions);
 
+	const ref = useRef<HTMLDivElement>(null);
+	useEffect(() => {
+		if (scrollToItem === item.itemId) {
+			ref.current?.scrollIntoView({ behavior: 'smooth' });
+			setScrollToItem(null);
+		}
+	});
```

Comment:
there should be a dependency array parameter here, something like this:
`}, [scrollToItem, item.itemId, setScrollToItem])`

(eslint will show errors if something is missing)

Reply:
Sure. I remember removing the dependency array at some point because the linter was screaming at me, but with the current code it works.